src/translator_en.h

```diff
@@ -2563,6 +2563,8 @@ class TranslatorEnglish : public Translator
         result+="the namespaces they belong to:";
       return result;
     }
+    virtual QCString trDefinition()  { return "Definition:";}
```

Comment:
Not sure it is a good idea to include the colon at the end in the translation text.